interfaces/IBF-dashboard/src/app/pages/dashboard/dashboard.page.scss

```diff
@@ -66,7 +66,8 @@ app-export-view {
           .component-title-thumbnail {
             height: 24px;
             width: 24px;
-            display: none;
+            display: block;
+            background-image: url('../../../assets/icons/alert-white.svg');
```

Comment:
@jannisvisser do we need to show a warning icon if there is no trigger?

Reply:
@gulfaraz good point.. I didn't think about it, just turned it from white to black.. I'll remove it then.